temporary.py

```diff
@@ -13,21 +16,23 @@
 fl = figlist_var()
 fl.next("freq")
 fl.image(d.C.setaxis("power", "#").set_units("power", "scan #"))
-d = d['t2':(-3e3,3e3)]
 d.ift("t2")
 fl.next("time")
 fl.image(d.C.setaxis("power", "#").set_units("power", "scan #"))
 d.register_axis({'t2':0})
 #d.ft('t2')
 #d = d['t2':(-3e3,3e3)]
 #d.ift('t2')
-best_shift, window_size = hermitian_function_test(d["ph1", 1]["ph2", 0], fl=fl)
+best_shift, window_size = hermitian_function_test(d["ph1", 1]["ph2", 0])
+logger.info(strm("hermitian best shift",best_shift))
 best_shift = 0.33e-3
```

Comment:
doesn't this just reset the best_shift so you aren't using the one the hermitian function test returns?

Reply:
I'm not using it, b/c even with the changes, the result is not great, and while I think my changes are in the right direction, I didn't have time to debug further